interoperator/pkg/webhook/default_server/sfserviceinstance/mutating/create_webhook.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2018 The Service Fabrik Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mutating
```

Comment:
better naming

Reply:
The `mutating` package name is used by kubebuilder too ([example](https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/gopath/src/project/pkg/webhook/default_server/firstmate/mutating/webhooks.go#L17)). I couldn't think of a better name, let me know if you have any recommendation.